django-verdant/rca/templates/rca/blocks/testimonial_block.html

```diff
@@ -1,9 +1,22 @@
 {% load wagtailimages_tags %}
 
-{% image value.image original %}
+<div class="testimonial__slide">
+    <div class="testimonial__content">
+        {% image value.image fill-600x335 class="testimonial__image" %}
 
-<p>{{ value.quote }}</p>
+        <blockquote class="testimonial__quote">
+            <p class="testimonial__quote-text">
+                <span class="testimonial__quote--highlight">
+                    <svg class="testimonial__quote-icon">
```

Comment:
You can add an `aria-hidden="true"` here, as the quote mark is purely decorative.